bl-plugins/rss/plugin.php

```diff
@@ -68,9 +74,9 @@ private function createXML()
 				$page = new Page($pageKey);
 				$xml .= '<item>';
 				$xml .= '<title>'.$page->title().'</title>';
-				$xml .= '<link>'.$page->permalink().'</link>';
+				$xml .= '<link>'.$this->urlRSS($page->permalink()).'</link>';
 				$xml .= '<description>'.Sanitize::html($page->contentBreak()).'</description>';
-				$xml .= '<pubDate>'.$page->date(DATE_RSS).'</pubDate>';
+				$xml .= '<pubDate>'.date('r',strtotime($page->getValue('dateRaw'))).'</pubDate>';
```

Comment:
It's not better to use the constant for RSS ? `DATE_RSS`

Reply:
Instead 'r' something like `date(DATE_RSS,strtotime($page->getValue('dateRaw')))`?

Reply:
The method `$page->date()` do the format for you. https://github.com/bludit/bludit/blob/master/bl-kernel/pagex.class.php#L107

And the constant `DATE_RSS` has the structure for RSS dates, is defined in PHP.

I don't get it why we need to change the format to `r`.

Reply:
This is synonym. I have changed it to `DATE_RSS`.